code/api/src/modules/user/model.js

```diff
@@ -19,7 +19,9 @@ module.exports = function(sequelize, DataTypes) {
 
   User.associate = function(models) {
     User.hasMany(models.Subscription)
```

Comment:
joins relationship, belongs to many requires the through to connect it to joins table
the as in the has many connect the userProducts to user in the resolver